### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by ging-dev**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          go build go-ddoser.go
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "latest"
+          prerelease: true
+          title: "Development Build"
+          files: |
+            go-ddoser
```
Comment (ging-dev) [inline .github/workflows/release.yml:28]: Run GA to do automatic packing

File: README.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
+Denial of Service Attacks Using Golang
+
+### Usage
+<img src="https://imgur.com/aARCzV9.png" align="right" height="100">
 
-### Set up commands
 ```bash
 git clone https://github.com/kenegdane/go-ddoser
 cd go-ddoser
-go get "github.com/gamexg/proxyclient"
```
Comment (ging-dev) [inline README.md:7]: may not need this line

**Comment by ciricode** [comment]
Use `runtime.NumCPU()` to get the exact number of CPU.
https://github.com/kenegdane/go-ddoser/blob/4062261326d4c2df7c9d242ba54b02e22b6294aa/go-ddoser.go#L153

**Comment by ging-dev** [comment]
> Use `runtime.NumCPU()` to get the exact number of CPU.
> 
> https://github.com/kenegdane/go-ddoser/blob/4062261326d4c2df7c9d242ba54b02e22b6294aa/go-ddoser.go#L153

Thanks, in general I have no knowledge of this stuff :/

**Comment by ging-dev** [comment]
@ciricode is it correct?

**Review by ciricode — APPROVED** [review]

**Pull request merged**